docs/modules/publisher-dmmigration.md

```diff
@@ -0,0 +1,143 @@
+[Home](../../readme.md)
+
+# Migration path
+
+The latest version 2.8.2 adds support for configuration via direct methods. OPC Publisher implements multiple [IoT Hub Direct Methods](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-direct-methods) which can be called from an application leveraging the [IoT Hub Device SDK](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-sdks). This document provides the migration path from 2.5.x to 2.8.2.
+
+The direct methods' request payload of version 2.8.2 is backwards compatible with the 2.5.x direct methods. The payload schema allows configuration of additional extensions introduced in the pn.json in the publisher 2.6.x and newer e.g. DataSetWriterGroup, DataSetWriterId, QueueSize per node, etc.
+
+For these set of methods, the encoding shall be JSON and no compression or payload chunking mechanism shall be applied in order to ensure the backwards compatibility with the 2.5.x version of OPC Publisher module.  
+
+
+
+### Direct Methods of version 2.5.x ###
```

Comment:
Inconsistent heading. Previous heading had one `#`, so this one should have two of them and not immediately three.